code/game/objects/items.dm

```diff
@@ -914,13 +914,13 @@ GLOBAL_DATUM_INIT(welding_sparks, /mutable_appearance, mutable_appearance('icons
 	if(rankName in special)
 		return rankName
 
+	if(rankName in job_icons)
+		return rankName
+
 	// Check if the job has a hud icon
 	if(assignmentName in job_icons)
 		return assignmentName
```

Comment:
Why switch these around? Assignment is alt-titles, where rank is actual job. If theres an assignment icon, it should be preferred over the rank icon imo

Reply:
I've switched them around again. This does mean that it is not possible to have a different icon for an existing job title (Eg. If your job title is Blueshield you can't change the HUD icon to be the clown icon)